package-lock.json

```diff
@@ -5,11 +5,13 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "essote-management-portal",
```

Comment:
Why package-lock has been updated? This PR does not have any changed to package.json so I would assume that also the lock-file should be untouched